codespeed/views.py

```diff
@@ -424,11 +446,20 @@ def timeline(request):
         defaultequid = data['equid']
     else:
         defaultequid = "off"
+    if 'quarts' in data:
+        defaultquarts = data['quarts']
+    else:
+        defaultquarts = "on"
+    if 'extr' in data:
+        defaultextr = data['extr']
+    else:
+        defaultextr = "on"
 
     # Information for template
     executables = {}
     for proj in Project.objects.filter(track=True):
         executables[proj] = Executable.objects.filter(project=proj)
+    use_median_bands = hasattr(settings, 'USE_MEDIAN_BANDS') and settings.USE_MEDIAN_BANDS
```

Comment:
I don't think `hasattr` is needed since you added `USE_MEDIAN_BANDS` to the basic `settings`